components/exercises/chapter-3-oscar/Minimum.test.tsx

```diff
@@ -0,0 +1,30 @@
+import { render, screen } from '@testing-library/react';
+import Minimum from './Minimum';
+
+describe('Minimum component', () => {
+  it('renders the name of the youngest person', () => {
```

Comment:
Incorrect. Since we haven't pressed the button yet, both people are shown.

Can we update these tests so they assert:
- Initially, both people are shown
- When the button is pressed, only the youngest is shown

Reply:
added